state-chain/TROUBLESHOOTING.md

```diff
@@ -77,7 +77,7 @@ Make sure to add `my-pallet/try-runtime` in the runtime's Cargo.toml, otherwise
 ### Compile the node with benchmark features enabled
 
 ```bash
-cargo cf-build-with-benchmarks
+cargo cf-build
```

Comment:
From what i understand, the `benchmark.sh` script uses the release build, so this cmd is not the right one. I can't see a cargo alias for building release with benchmarks. What cmd do ppl use for this case?

Reply:
There's no need for you to run the specific benchmarks anyway, just get something that compiles. The real benchmarks will be run on reference hardware within a CI job

Reply:
> There's no need for you to run the specific benchmarks anyway, 

But I need to use a new weight function in `on_idle`, It will not build until the weight file is updated, so i need to run the benchmark manually so i can continue work.

> the real benchmarks will be run on reference hardware within a CI job

When does this happen? Am I supposed to have the updated weight file in my PR?

I am still unsure about the procedure around the weight files. This troubleshooting guide does not explain much, and it is out of date, the commands don't exist.

Reply:
No need to update the weights file in the PR, it happens as part of a release.